specification/compute/resource-manager/Microsoft.Compute/preview/2021-06-01-preview/examples/GetDiagnosticOperation.json

```diff
@@ -0,0 +1,24 @@
+{
+    "parameters": {
+      "subscriptionId": "{subscription-id}",
+      "location": "{location-value}",
+      "operationId": "{operation-Id}"      
+    },
+    "responses": {
+        "200": {
+            "headers": {
+                "azure-asyncoperation": "Succeeded",
```

Comment:
Azure-AsyncOperation header should be a absolute URI.
Please refer rpc async guidelines here: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md

Reply:
This operation outputs the final-state or the polling url via "location" header. As per http://azure.github.io/autorest/extensions/#x-ms-long-running-operation, the value could be one of "azure-async-operation" or "location" or "original-uri". Will remove the additional header - "azure-async-operation". Please advise.

Reply:
sounds good.